### PR TITLE
adds the funny hand mirror to cafe costuming kit

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
adds the syndicate hand mirror to the costuming kit

## How This Contributes To The Skyrat Roleplay Experience

I want to change my hair without killing myself

## Changelog

:cl:
qol: Added a hand mirror to the cafe costume box, so you can change your hair.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/ghostcafe/code/ghost_role_spawners.dm [hunk 1]
```diff
@@ -109,6 +109,7 @@
 	new /obj/item/storage/backpack/chameleon(src)
 	new /obj/item/storage/belt/chameleon(src)
 	new /obj/item/card/id/advanced/chameleon(src)
+	new /obj/item/hhmirror/syndie
```
Comment (larentoun) [inline modular_skyrat/modules/ghostcafe/code/ghost_role_spawners.dm]: Don't you need to specify the loc?
```suggestion
	new /obj/item/hhmirror/syndie(src)
```

**Comment by desertmote** [comment]
does this actually function? last time i checked using a hhmirror it would give me the message to go visit a barber.

**Comment by YakumoChen** [comment]
> does this actually function? last time i checked using a hhmirror it would give me the message to go visit a barber.

The syndicate one is supposed to work by itself

**Comment by tf-4** [comment]
_Does_ it work by itself, though?

**Comment by YakumoChen** [comment]
that would be a great thing to test but it wouldn't be something this PR would be able to fix, or relevant to the PR

**Review by tf-4 — DISMISSED** [review]
Whether or not the item you are adding is functional is something you should already be aware of.
I shouldn't have to be asking "does this PR achieve what it says it does".

Test it.

**Comment by YakumoChen** [comment]
@tf-4 the mirror works, as it always did

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**